<?xml version="1.0"?>
<launch>
	<rosparam ns="color_range" command="load" file="$(find zeabus_vision_sk)/params/color_top.yaml">
	</rosparam>
	<node pkg="zeabus_vision_sk" name="color_range_top" type="vision_top.py" output="screen">
		<param name="topic_top" value='/leftcam_top/image_raw/compressed'/>
		<!--<param name="topic_top" value='/rightcam_top/image_raw/compressed'/>-->
	</node>
</launch>
